src/Schema/IndexDefinition.php

```diff
@@ -5,11 +5,20 @@
 use Illuminate\Support\Fluent;
 
 /**
- * @method $this interleave(string $table)
+ * @method $this interleaveIn(string $table)
  * @method $this nullFiltered()
  * @method $this storing(string[] $columns)
  * @extends Fluent<string, mixed>
  */
 class IndexDefinition extends Fluent
 {
+    /**
+     * @deprecated use interleaveIn instead.
```

Comment:
It is `@deprecated` but does not seem to work correctly when called.
I changed the test code to try and run it, but it failed.

```diff
diff --git a/tests/Schema/BlueprintTest.php b/tests/Schema/BlueprintTest.php
index a47089b..f2fffbe 100644
--- a/tests/Schema/BlueprintTest.php
+++ b/tests/Schema/BlueprintTest.php
@@ -424,7 +424,7 @@ class BlueprintTest extends TestCase
         $conn = $this->getDefaultConnection();
 
         $blueprint = new Blueprint('UserItem', function (Blueprint $table) {
-            $table->index(['userId', 'createdAt'])->interleaveIn('User');
+            $table->index(['userId', 'createdAt'])->interleave('User');
         });
 
         $queries = $blueprint->toSql($conn, new Grammar());
```

```
1) Colopl\Spanner\Tests\Schema\BlueprintTest::test_index_with_interleave
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'create index `useritem_userid_createdat_index` on `UserItem` (`userId`, `createdAt`), interleave in `User`'
+'create index `useritem_userid_createdat_index` on `UserItem` (`userId`, `createdAt`)'

/project/tests/Schema/BlueprintTest.php:433
```

It delegates to `interleaveIn()`, but the name seems to be processed as `interleave` instead of `interleaveIn`.